examples/parser.js

```diff
@@ -18,7 +22,7 @@ fastify.addContentTypeParser('application/jsoff', function (request, payload, do
 fastify.addContentTypeParser('application/x-www-form-urlencoded', function (request, payload, done) {
   let body = ''
   payload.on('data', function (data) {
-    body += data
+    body += data.toString()
```

Comment:
hmm. I guess urlencoded si pure ascii, but if it is multibyte encoded, than this bodyparser could be error prone.

Should be imho:

```js
fastify.addContentTypeParser('application/x-www-form-urlencoded', function (request, payload, done) {
  const body = []
  payload.on('data', function (data) {
    body.push(data)
  })
  payload.on('end', function () {
    try {
      const parsed = querystring.parse(Buffer.concat(body).toString())
      done(null, parsed)
    } catch (e) {
      done(e)
    }
  })
  payload.on('error', done)
})

```